### ISSUE TITLE
New-IntersightBulkMoCloner throws Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by davidstanaway** [CiscoDevNet/intersight-powershell#147]
**Describe the bug**
Using New-IntersightBulkMoCloner to clone an IntersightServerProfileTemplate to an IntersightServerProfile throws an error:
Object reference not set to an instance of an object.

**To Reproduce**
Steps to reproduce the behavior
Using examples/server/DeriveServerProfileFromTemplate.ps1
but with our own server profile template and org values, no server profile is derived, or object returned and the command throws an error.

$source = Get-IntersightServerProfileTemplate -Name "server_template"
$orgRef = Get-IntersightOrganizationOrganization -Name default
$targetProfile = Initialize-IntersightServerProfile -Name cloned-server-profile -Description "cloned from template" -Organization (Get-IntersightMoMoRef -ManagedObject ($orgRef))
New-IntersightBulkMoCloner -Sources $source -Targets $targetProfile

**Version used**
 - Intersight PowerShell Module version? git/main 
 - Intersight Appliance Version (if using an Intersight Appliance as target) | intersight.com

**Expected behavior**

I expect a server profile to be created called in above example cloned-server-profile the same as if I had derived profiles from the server profile template named "server_template" in above example, in the organization default.

**Screenshots**
![image](https://github.com/CiscoDevNet/intersight-powershell/assets/37458016/c193341f-25e4-4e34-8fb5-a99b78eb8468)

**Additional context**

**Comment by davidstanaway** [comment]
This is probably tied to https://github.com/CiscoDevNet/intersight-powershell/issues/139

**Comment by davidstanaway** [comment]
> This is probably tied to #139

Can't role back to that version as we are using New-IntersightVnicVnicTemplate which is a new feature.

**Comment by tgrabrian** [comment]
Any updates on this one? Currently running into this bug. It was indeed working just fine on previous versions of this PowerShell module.

**Comment by Ghufz** [comment]
This issue has been fixed in Intersight.PowerShell module version 1.0.11.17956.

This issue also requires a backend fix, which will be available in production by in coming weeks.

As a workaround for the backend issue, The New-IntersightBulkMoCloner cmdlet can use with the -json switch parameter.

```powershell
$src = Initialize-IntersightMoBaseMo -Moid xxxxxxxxxxxxxxxx -ClassId ServerProfileTemplate -ObjectType ServerProfileTemplate
$targ = Initialize-IntersightServerProfile -Name ps_cloned_profile
$result = New-IntersightBulkMoCloner -Sources @($src) -Targets @($targ) -Json | ConvertFrom-json


```

**Comment by Ghufz** [comment]
Cmdlet is working as expected in released version 1.0.11.18012
